tests/integration-tests/utils.py

```diff
@@ -38,6 +39,42 @@
 }
 
 
+@dataclass
+class CWMetric:
```

Comment:
minor: we could move these new classes/methods into a separate cloudwatch_utils.py